lib/logger/utils.ts

```diff
@@ -103,7 +103,10 @@ export function sanitizeValue(
   seen = new WeakMap<NestedValue, unknown>()
 ): any {
   if (is.string(value)) {
-    return sanitize(value);
+    return sanitize(value)
+      .split(/\r?\n/)
+      .map((line) => line.split(' ').map(sanitizeUrl).join(' '))
```

Comment:
Did you generally see URLs separated by spaces on either side? I'm wondering if we are likely to see commas or opening/closing brackets ()

Reply:
🤔 not thought about that. back to draft